### ISSUE TITLE
Using generic image to run older NAV & mapped folders

### ISSUE THREAD

**Issue by Koubek** [Microsoft/nav-docker#146]
I think I have found a small issue. 
+ I am using **generic** image to run NAV 2015.
+ I am using **volumes** to share add-ins between the container and the host system (`-v "$sharedMasterFolder\$hostname\Add-ins:C:\Program Files\Microsoft Dynamics NAV\$navVersionDir\Service\Add-ins\Docker-Share"`).
+ You are using `C:\Program Files\Microsoft Dynamics NAV` to control the flow of the code (https://github.com/Microsoft/nav-docker/blob/828ef895cbd4f5f5546dff7f7f7d9a796dda5995/Run/start.ps1#L76).

When I map my add-ins folder, the symlink creates `C:\Program Files\Microsoft Dynamics NAV\...` and the program flow changes incorrectly.

**Comment by freddydk** [comment]
You are right - I should probably check for existence of the service tier .exe file instead

**Comment by freddydk** [comment]
Next version will have line 76 replaced with

if (!(Test-Path "C:\Program Files\Microsoft Dynamics NAV\*\Service\*.exe" -PathType Leaf)) {

**Comment by Koubek** [comment]
OK, this should be OK. I did something slightly different (and I can confirm it works as I use it on some instances):
```PowerShell
if ((Get-ChildItem 'C:\Program Files\Microsoft Dynamics NAV\*\Microsoft.Dynamics.Nav.Server.exe' -Recurse | Measure-Object).Count -eq 0) {
```

**Comment by freddydk** [comment]
is included in 0.0.4.5

**Comment by Koubek** [comment]
Thanks!
